README.md

```diff
@@ -9,48 +9,85 @@ reMarkable screen sharing over SSH.
 
-1. Install [git for windows](https://gitforwindows.org/), which includes `Git BASH`.
-2. Install [ffmpeg for windows](https://ffmpeg.org/download.html#build-windows).
-3. Download [lz4 for windows](https://github.com/lz4/lz4/releases) and extract the `zip` to a folder where you'll remember it (e.g. `C:\Users\{username}\lz4`).
-4. Add the `ffmpeg` **and** `lz4` directories to the windows `Path` environment. [Here is a quick guide how.](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)
+1. Install [ffmpeg for windows](https://ffmpeg.org/download.html#build-windows).
+2. Download [lz4 for windows](https://github.com/lz4/lz4/releases) and extract the `zip` to a folder where you'll remember it (e.g. `C:\Users\{username}\lz4`).
+3. Add the `ffmpeg` **and** `lz4` directories to the windows `Path` environment. [Here is a quick guide how.](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)
     - Control Panel > Edit the system environment variables > Environment Variables
     - Find the `Path` variable under System variables, click edit.
     - Add two _New_ entries: one to the **bin** directory in the `ffmpeg` directory, and one to the `lz4` directory you created.
     - Click OK
-5. (Re)start `bash` so the new `Path` is used.
-6. Generate a new ssh key using `ssh-keygen`.
-7. Send the public key to the reMarkable (connect trough USB cable) using `ssh-copy-id -i ~/.ssh/id_rsa root@10.11.99.1`
-8. Try out `ssh root@10.11.99.1`, it should **not** prompt for a password.
-9. Go to step **3** of [the Unix installation instructions](#Unix).
+4. (Re)start `bash` so the new `Path` is used.
+5. Generate a new ssh-key using `ssh-keygen`.
+6. Send the public key to the reMarkable (connect trough USB cable) using `ssh-copy-id -i ~/.ssh/id_rsa root@10.11.99.1`
+7. Try out `ssh root@10.11.99.1`, it should **not** prompt for a password.
+
+### reStream installation
+
+We need to download the [latest release](https://github.com/rien/reStream/releases/) files.  
+In particular, `reStream.sh` is the executable on the host, and `restream.arm.static` is the binary, which have to be moved on the reMarkable with the name `restream`.
+
+#### Host
+
+Download [reStream.sh](https://github.com/rien/reStream/releases/download/1.1/reStream.sh) and make it executable
+
+```
+$ chmod +x reStream.sh
+```
+> **Tip**  
+> If you save `reStream.sh` in a `PATH` directory as `reStream`, you can launch it as `reStream`.  
+> On Ubuntu, list these folders with `echo $PATH`. One should be`/usr/local/bin`.  
+> As root, download the executable there: 
+```
+# wget https://github.com/rien/reStream/releases/download/1.1/reStream.sh -O /usr/local/bin/reStream
+# chmod +x /usr/local/bin/reStream
+```
+
+#### reMarkable
+
+You can install `restream` on reMarkable in three ways.
+
+1. Download the [restream](https://github.com/rien/reStream/releases/download/1.1/restream.arm.static) binary onto your host, move it to reMarkable and make it executable.
```

Comment:
I would use an unordered (bulleted) list instead of numbers, it looks like multiple steps now.

I would also reverse the order:
- toltec
- direct download
- host dowload

This way the easiest method comes first.

Reply:
You are right. Bullets are better.
I do not agree about the order. I think adding unnecessary dependencies is not a good tip.

Reply:
My reasoning is not that users will install toltec, but that they will use toltec if it's already installed.